ResearchKit/Common/ORKOrderedTask_Internal.h

```diff
@@ -36,8 +36,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface ORKOrderedTask ()
```

Comment:
Wasn't sure about that. It isn't being used but I didn't know if there might be other devs who had added something to that file. I can remove it. :)